backend/fms_core/viewsets/full_sample.py

```diff
@@ -0,0 +1,235 @@
+                                                               WHERE id IN %s
+                                                               UNION ALL
+                                                               SELECT child.id, child.location_id
+                                                               FROM fms_core_container AS child, parent
+                                                               WHERE child.location_id = parent.id
+                                                           )
+                                                           SELECT * FROM parent''', params=[container_ids])
+
+            return self.queryset.filter(container__in=parent_containers)
+
+        return self.queryset
+
+
+    def create(self, request, *args, **kwargs):
+        error = {}
+        full_sample = request.data
+
+        biosample_data = dict(
+            collection_site=full_sample['collection_site'],
+            **(dict(individual_id=full_sample['individual']) if full_sample['individual'] is not None else dict()),
+            **(dict(alias=full_sample['alias']) if full_sample['alias'] is not None else dict()),
+        )
+
+        try:
+            biosample = Biosample.objects.create(**biosample_data)
+
+            derived_sample_data = dict(
+                biosample_id=biosample.id,
+                sample_kind_id=full_sample['sample_kind'],
+                **(dict(tissue_source=full_sample['tissue_source']) if full_sample['tissue_source'] is not None else dict()),
+            )
+            if full_sample['experimental_group']:
+                derived_sample_data['experimental_group'] = json.dumps([
+                    g.strip()
+                    for g in RE_SEPARATOR.split(full_sample['experimental_group'])
+                    if g.strip()
+                ])
+
+            derived_sample = DerivedSample.objects.create(**derived_sample_data)
+
+            sample_data = dict(
+                name=full_sample['name'],
+                volume=full_sample['volume'],
+                creation_date=full_sample['creation_date'],
+                container_id=full_sample['container'],
+                **(dict(comment=full_sample['comment']) if full_sample['comment'] is not None else dict()),
+                **(dict(coordinates=full_sample['coordinates']) if full_sample['coordinates'] is not None else dict()),
+                **(dict(concentration=full_sample['concentration']) if full_sample['concentration'] is not None else dict()),
+            )
+
+            sample = Sample.objects.create(**sample_data)
+
+            DerivedBySample.objects.create(derived_sample_id=derived_sample.id,
+                                           sample_id=sample.id,
+                                           volume_ratio=1)
+
+        except ValidationError as err:
+            raise ValidationError(err)
+
+        #Serialize created sample
+        serializer = SampleSerializer(sample)
```

Comment:
you should get the full sample and serialize it instead of patching in the missing fields. It will break as soon as we add a field or remove one.